schema/field.go

```diff
@@ -941,7 +941,7 @@ func (field *Field) setupValuerAndSetter() {
 						case *sql.RawBytes:
 							bytes = []byte(*t)
 						default:
-							panic("wrong type")
+							return
```

Comment:
It is too generic. 

Plus I do not get how (type) of the object can be nil? 
Why it happens for OAS PR?

Reply:
Because in `openapi3.Swagger` struct, there are struct pointers like `*Contact`, `*License` etc. they can be nil.

Reply:
I see.

OK, it can work, I guess.

Reply:
Can we then add more specific check for Nil case?

Reply:
@buger I added a `case nil:`. Is that ok?